tests/integration/test_materialize_mysql_database/test.py

```diff
@@ -16,7 +16,8 @@
 
 node_db_ordinary = cluster.add_instance('node1', user_configs=["configs/users.xml"], with_mysql=False, stay_alive=True)
 node_db_atomic = cluster.add_instance('node2', user_configs=["configs/users_db_atomic.xml"], with_mysql=False, stay_alive=True)
-
+node_disable_bytes_settings = cluster.add_instance('node3', user_configs=["configs/users_disable_bytes_settings.xml"], with_mysql=False, stay_alive=True)
+node_disable_rows_settings = cluster.add_instance('node4', user_configs=["configs/users_disable_rows_settings.xml"], with_mysql=False, stay_alive=True)
```

Comment:
This is not needed if these two settings are disabled by default.

Reply:
This is needed even though the value is disabled. Because I need to sure when someone uses these settings that are valid.